app/Http/routes.php

```diff
@@ -31,9 +31,13 @@
         return 'Rogue API version 1';
     });
 
-    Route::get('reportbacks', 'Api\ReportbackController@index');
-    Route::post('reportbacks', 'Api\ReportbackController@store');
+    // /items
     Route::put('items', 'Api\ReportbackController@updateReportbackItems');
+
+    // /reactions
     Route::post('reactions', 'Api\ReactionController@store');
-    Route::get('users', 'UsersController@index');
```

Comment:
@katiecrane I don't think it does! Just hits the `index` function in `UsersController` which returns `list of all users` - I guess I can get rid of this too to clean up!